Dockerfile

```diff
@@ -60,3 +60,4 @@ USER t
 WORKDIR /src
 RUN Rscript install.R --local
 CMD ["/src/tests/runtest"]
+CMD ["/src/tests/runcheck"]
```

Comment:
This _overwrites_ the previous CMD rather than running in addition to it.

Reply:
Oh! Wasn't aware of that. Will have to add it to the `runtest` script then.